src/mgmt/cli_mgmt.c

```diff
@@ -197,7 +197,7 @@ free_get_attach_info_resp(Mgmt__GetAttachInfoResp *resp)
 	mgmt__get_attach_info_resp__free_unpacked(resp, &alloc.alloc);
 }
 
-static void
+void
```

Comment:
instead of this, you probably want to create wrapper functions such as dc_put_attach_info() dc_get_attach_info() that would be publically exposed and call static functions from those wrappers